_posts/2023-02-20-intelligent-automation.md

```diff
@@ -0,0 +1,223 @@
+We need to answer it with a digital transformation of the current landscape in organizations, and intelligent automation aims for this lofty goal by enhancing the traditional business process management discipline with process automation technology, guided by business rules and bots (both RPA and AI). 
+The way to successfully monitor whether the path taken will lead to success is to oversee this process automation with the proper information governance.
+
+Intelligent Automation thus plays on at least four aspects of the organization:
+1.	Adding intelligence to (operational) processes.
+2.	Augmenting decision-making within a process through analytics.
+3.	Monitoring processes for correct operation and adapting them to meet changes in the strategic or tactical direction of the organization.
+4.	Applying intelligent automation and analytics to strategic and tactical decision-making.
+
+The introduction of bots on top of the typical iBPMS is rooted in the idea that human interaction in processes is still needed at certain times, and if our RPA bots can use the same interfaces as their human counterparts, there would be an efficiency gain by alleviated work from knowledge workers and a cost gain by eliminating the need to implement specific interfaces for these bots. 
+It is however imperative that these bots know exactly what to do, and thus the same rules that guide human participants in the process need to apply to these bots. 
+In addition to this, the same level of transparency (such as for audit or privacy purposes) should also apply to the bots. 
+This way correct optimization of the process automation can be determined.
+
+These different components are aptly assembled in Nathaniel Palmer’s vision for future process automation solutions:
+{:refdef: style="text-align: center;"}
+<img src="{{ '/img/2023-02-20-intelligent-automation/ibpms.png' | prepend: site.baseurl }}" alt="Workspace" class="image fit" style="margin:0px auto; max-width:100%">
+{: refdef}
+
+The three guiding principles are Rules, Robots, and Relationships. 
+The Robot part of the equation was already tackled earlier in this post. 
+The Relationship part focuses mainly on the data used in the processes. 
+There is a shift here from the problems of storing and replication data locally towards the problems of where to get data in the wide world. 
+The different pieces of the information puzzle could be spread across dozens of repositories all over the world, and although our complete information metamodel still exists, its parts need to be fetched from each of these repositories to complete any sort of 360° view.
+
+Rules are formed by automating decisions and letting AI solutions approach the available information with machine learning to perform both analytical and predictive analyses. 
+This isn’t magic, however, and these AI need to be properly trained using historical data to perform with any level of adequacy. 
```

Comment:
```suggestion
This isn’t magic, however, and these AI's need to be properly trained using historical data to perform with any level of adequacy. 
```

Not sure if AI can be plural @peterdekinder ?

Reply:
It's like 'fish'. Singular and plural are the same.